README.md

```diff
@@ -187,74 +192,136 @@ tx_response = send_reliable_submission(my_tx_payment_signed, client)
 
-```pycon
->>> from xrpl.ledger import get_fee
->>> fee = get_fee(client)
->>> print(fee)
-10
+```py
+from xrpl.ledger import get_fee
+fee = get_fee(client)
+print(fee)
+# 10
 ```
 
 #### Auto-filled fields
 
-The `xrpl-py` library automatically populates the `fee` and `last_ledger_sequence` fields when you create transactions. In the example above, you could omit those fields and let the library fill them in for you.
-
-```pycon
->>> from xrpl.models.transactions import Payment
->>> from xrpl.transaction import send_reliable_submission
->>> # prepare the transaction
->>> # the amount is expressed in drops, not XRP
->>> # see https://xrpl.org/basic-data-types.html#specifying-currency-amounts
->>> my_tx_payment = Payment(
->>>     account="rMPUKmzmDWEX1tQhzQ8oGFNfAEhnWNFwz",
->>>     amount="2200000",
->>>     destination="rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
->>>     sequence=test_wallet.sequence,
->>> )
->>> # sign the transaction
->>> my_tx_payment_signed = safe_sign_transaction(my_tx_payment, test_wallet)
->>> # submit the transaction
->>> tx_response = send_reliable_submission(my_tx_payment_signed, client)
->>> my_tx_payment
-Payment(
-    account='rMPUKmzmDWEX1tQhzQ8oGFNfAEhnWNFwz',
-    transaction_type=<TransactionType.PAYMENT: 'Payment'>,
-    fee=10000,
-    sequence=16034065,
-    account_txn_id=None,
-    flags=0,
-    last_ledger_sequence=10268600,
-    memos=None,
-    signers=None,
-    source_tag=None,
-    signing_pub_key=None,
-    txn_signature=None,
-    amount='2200000',
-    destination='rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe',
-    destination_tag=None,
-    invoice_id=None,
-    paths=None,
-    send_max=None,
-    deliver_min=None
+The `xrpl-py` library automatically populates the `fee`, `sequence` and `last_ledger_sequence` fields when you create transactions. In the example above, you could omit those fields and let the library fill them in for you.
+
+```py
+from xrpl.models.transactions import Payment
+from xrpl.transaction import send_reliable_submission, safe_sign_and_autofill_transaction
+# prepare the transaction
+# the amount is expressed in drops, not XRP
+# see https://xrpl.org/basic-data-types.html#specifying-currency-amounts
+my_tx_payment = Payment(
+    account=test_wallet.classic_address,
+    amount="2200000",
+    destination="rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe"
 )
+# sign the transaction with the autofill method
+# (this will auto-populate the fee, sequence, and last_ledger_sequence)
+my_tx_payment_signed = safe_sign_and_autofill_transaction(my_tx_payment, test_wallet, client)
+# submit the transaction
+tx_response = send_reliable_submission(my_tx_payment_signed, client)
+my_tx_payment
```

Comment:
1. Isn't it `print(my_tx_payment_signed)` instead of `my_tx_payment` here?
2. It might make more sense to place it before `# submit the transaction` (line 220)?